### PR TITLE
Fix a bunch of internal links

### PR DESCRIPTION
**Pull request by limemloh**
Fixes most of the internal links except the ones in `downloads.rst`, which are be fixed by @jens-concordium in another branch.

### CODE REVIEW

File: source/testnet/references/concordium-client.rst [hunk 1]
```diff
@@ -45,7 +30,7 @@ directory:
 **This page will describe the commands that are related with configuration of
 the client, but the rest of available commands will be discussed on the pages
 where the features that use them are documented. Some commands will perform**\
-`queries`_\ **and others send** :ref:`transactions<transactions>`\ **.**
+:ref:`queries<testnet-query-node>`\ **and others send** :ref:`transactions<transactions>`\ **.**
```
Comment (jens-concordium) [inline source/testnet/references/concordium-client.rst:33]: Hele den her paragraf er i bold. Ved ikke om det er meningen, eller om det er en fejl der er sket nu

Comment (jens-concordium) [reply source/testnet/references/concordium-client.rst:33]: Der mangler også et par mellemrum rundt om query linket

Comment (abizjak) [reply source/testnet/references/concordium-client.rst:33]: Could we please stick to English.

Comment (jens-concordium) [reply source/testnet/references/concordium-client.rst:33]: Ooops, yes ofcourse. That was a brainfart.

Comment (limemloh) [reply source/testnet/references/concordium-client.rst:33]: I think it would fit better as a `..note::` rather than bold

**Review by jens-concordium — APPROVED** [review]

**Pull request merged**